py/torch_tensorrt/dynamo/conversion/aten_ops_converters.py

```diff
@@ -420,3 +420,501 @@ def aten_ops_clone(
         name,
         args[0],
     )
+
+
+@dynamo_tensorrt_converter(torch.ops.aten.exp.default)  # type: ignore[misc]
+def aten_ops_exp(
+    network: TRTNetwork,
+    target: Target,
+    args: Tuple[Argument, ...],
+    kwargs: Dict[str, Argument],
+    name: str,
+) -> Union[TRTTensor, Sequence[TRTTensor]]:
+    input_val = args[0]
+    if (isinstance(input_val, TRTTensor)) and (
```

Comment:
Lets put the input validation in either impl.unary.exp or impl.unary.unary_op,  probably in the op specific part since it seems to vary between ops